_api-reference/cluster-health.md

```diff
@@ -105,4 +105,112 @@ The following table lists all response fields.
 ## Required permissions
 
 If you use the security plugin, make sure you have the appropriate permissions:
-`cluster:monitor/health`.
\ No newline at end of file
+`cluster:monitor/health`.
+
+## Cluster health by awareness attribute
+Introduced 2.5
+{: .label .label-purple }
+
+Cluster health by awareness attribute is an experimental feature. Therefore, we do not recommend the use of cluster health by awareness attribute in a production environment. For updates on the progress of cluster health by awareness attribute, or if you want leave feedback that could help improve the feature, refer to the [GitHub issue]().
+{: .warning}
+
+Previously, cluster health check could be performed only for the full cluster. Cluster health by awareness attribute lets you perform a cluster health check for a specified awareness attribute (for example, zone or rack). With cluster health by awareness attribute, shard allocation awareness attempts to separate primary and replica shards across multiple awareness attribute values. This partition gives you a closer look into a specific awareness attribute.
```

Comment:
Just a comment. This could be due to my unfamiliarity with the terminology.
"... shard allocation awareness ..."
Is there a way to word this concept differently? Like "... the awareness feature of shard allocation attempts to ..."
When I read this I string together "shard allocation awareness attempts" as a single thing, and the sentence then seems incomplete.

Reply:
Is "cluster health check" a proper name? Should it be set apart from regular typeface to indicate a proper name? Or, is it a common noun? In that case, I'd write "Previously, **a** cluster health check could..."

Reply:
> Just a comment. This could be due to my unfamiliarity with the terminology. "... shard allocation awareness ..." Is there a way to word this concept differently? Like "... the awareness feature of shard allocation attempts to ..." When I read this I string together "shard allocation awareness attempts" as a single thing, and the sentence then seems incomplete.

Or, "...the shard allocation awareness attribute...", which would be consistent with the phrasing "cluster health by awareness attribute."  The way I read this phrasing is that there are awareness attributes and "shard allocation" or "cluster health by" are types of awareness attributes.

Reply:
["shard allocation awareness"](https://opensearch.org/docs/latest/opensearch/cluster/#shard-allocation-awareness) is a term, so I cannot split it up.

Reply:
I can change "attempts" to "tries" if that makes a difference.

Reply:
I agree, "attempts" is the better word. I think the style issue we talked about above would probably solve this. In other words, if you set "shard allocation awareness" off in italics, or some other style, the eye would immediately separate the two and make sense of things. 
In the end, if you leave this as is, anyone who has my experience will probably just read it twice and figure it out anyhow.